crud/select.lua

```diff
@@ -233,9 +234,12 @@ function select_module.pairs(space_name, user_conditions, opts)
             error(string.format("Failed to get next object: %s", err))
         end
 
-        local obj, err = utils.unflatten(tuple, iter.space_format)
-        if err ~= nil then
-            error(string.format("Failed to unflatten next object: %s", err))
+        local obj = tuple
```

Comment:
Maybe it's `result`, not object? Or valiue.

`obj = tuple` confuses me, this feature should separate objects and tuples cases.

Reply:
Ok, fixed